### DISCUSSION TITLE
How to implement multi-physical link in granet3.0?

### DISCUSSION THREAD

**Discussion by qqli579** [gem5/gem5#726] [Q&A] [answered]
Whether multiple InternalLink are supported between two routers in garnet3.0. For example, there are two routers A and B. I want to establish three physical links from A to B. Is this possible?

**Comment by MeatBoy106** [comment, marked as answer]
It should be as simple as creating multiple links connecting A and B in the topology script and partitioning the supported vnets between these links.

You might be able to reproduce a similar behavior using a single link with well chosen width and vcs_pet_vnet link parameters. I didn't check the implementation details to support that claim though.

**Reply by qqli579** [reply]
As you said, it can be achieved. Use the supported_vnets parameter to configure it.
